PWGCF/EBYE/NetLambda/AliAnalysisTaskNetLambdaIdent.cxx

```diff
@@ -528,8 +528,8 @@ void AliAnalysisTaskNetLambdaIdent::UserExec(Option_t *){
 	      eta = mctrack->Eta();
 	      phi = mctrack->Phi();
 	      nd = mctrack->GetNDaughters();
-	      fd = mctrack->GetFirstDaughter();
-	      ld = mctrack->GetLastDaughter();
+	      fd = mctrack->GetDaughterFirst();
+	      ld = mctrack->GetDaughterFirst();
```

Comment:
Should this be `GetDaughterLast()`?

Reply:
Hi @akubera 

Right! My fault. I've just pushed a fix to the master. BTW, this class is using TParticle to access the MC track in ESDs, this will be forbidden in another transition (in a few weeks).

Cheers,
 Ruben